WREN compilation warnings on Ubutnu 18.04

When compiling WREN I have these warnings that should be fixed or suppressed:
```
PhongMaterial.cpp:54:88: warning: shift count >= width of type [-Wshift-count-overflow]
    return static_cast<size_t>(mCacheData->id() << 1) | (textureId << 16) | (programId << 32) |
```
```
PbrMaterial.cpp:69:88: warning: shift count >= width of type [-Wshift-count-overflow]
    return static_cast<size_t>(mCacheData->id() << 1) | (textureId << 16) | (programId << 32) |
         
```

--- a/src/wren/PbrMaterial.cpp
+++ b/src/wren/PbrMaterial.cpp
@@ -60,7 +60,7 @@ namespace wren {
   }
 
   size_t PbrMaterial::sortingId() const {
-    const size_t programId = static_cast<size_t>(mDefaultProgram->glName());
+    const unsigned long long programId = static_cast<unsigned long long>(mDefaultProgram->glName());
 
     size_t textureId = 0;
     if (mTextures[0].first)

--- a/src/wren/PhongMaterial.cpp
+++ b/src/wren/PhongMaterial.cpp
@@ -45,7 +45,7 @@ namespace wren {
   }
 
   size_t PhongMaterial::sortingId() const {
-    const size_t programId = static_cast<size_t>(mDefaultProgram->glName());
+    const unsigned long long programId = static_cast<unsigned long long>(mDefaultProgram->glName());
 
     size_t textureId = 0;
     if (mTextures[0].first)

